unitTests/ui/CategoryFacet/CategoryFacetTest.ts

```diff
@@ -10,25 +10,25 @@ import { IBuildingQueryEventArgs } from '../../../src/events/QueryEvents';
 import { first, range, pluck, shuffle, partition } from 'underscore';
 import { analyticsActionCauseList } from '../../../src/ui/Analytics/AnalyticsActionListMeta';
 
-export function CategoryFacetTest() {
-  function buildSimulateQueryData(numberOfResults = 11, numberOfRequestedValues = 11): ISimulateQueryData {
-    const fakeResults = FakeResults.createFakeResults();
-    const queryBuilder = new QueryBuilder();
-    fakeResults.categoryFacets.push(FakeResults.createFakeCategoryFacetResult('@field', [], 'value', numberOfResults));
-    queryBuilder.categoryFacets.push({
-      field: '@field',
-      path: pluck(fakeResults.categoryFacets[0].parentValues, 'value'),
-      maximumNumberOfValues: numberOfRequestedValues
-    });
-    return { results: fakeResults, query: queryBuilder.build() };
-  }
+export function buildSimulateCategoryFacetQueryData(numberOfResults = 11, numberOfRequestedValues = 11): ISimulateQueryData {
```

Comment:
Hmm, not sure about the name here. What about `buildCategoryFacetResults` or `createCategoryFacetResults`?